[qtinterfaceframework][CI] build failure

**Host Environment**
- OS: x64-windows

**To Reproduce**
https://dev.azure.com/vcpkg/public/_build/results?buildId=69334&view=logs&j=7922e5c4-0103-5f8f-ad17-45ce9bb98e80

**Failure logs**
```
[1/2] "D:/downloads/tools/cmake-3.22.2-windows/cmake-3.22.2-windows-i386/bin/cmake.exe" -E chdir "D:/buildtrees/qtinterfaceframework/x64-windows-rel" "D:/downloads/tools/cmake-3.22.2-windows/cmake-3.22.2-windows-i386/bin/cmake.exe" -S "D:/buildtrees/qtinterfaceframework/src/564c6bf227-ee28b9e83e.clean" "-DQT_USE_DEFAULT_CMAKE_OPTIMIZATION_FLAGS:BOOL=ON" "-DQT_BUILD_EXAMPLES:BOOL=OFF" "-DQT_BUILD_TESTS:BOOL=OFF" "-DQT_BUILD_BENCHMARKS:BOOL=OFF" "-DHOST_PERL:PATH=D:/downloads/tools/perl/5.32.1.1/perl/bin/perl.exe" "-DINSTALL_BINDIR:STRING=bin" "-DINSTALL_LIBEXECDIR:STRING=bin" "-DINSTALL_PLUGINSDIR:STRING=Qt6/plugins" "-DINSTALL_QMLDIR:STRING=Qt6/qml" "-DPython3_EXECUTABLE=D:/downloads/tools/python/python-3.10.2-x64/python.exe" "-DQT_SYNCQT:PATH=D:/installed/x64-windows/tools/Qt6/bin/syncqt.pl" "-DCMAKE_MAKE_PROGRAM=D:/downloads/tools/ninja/1.10.2-windows/ninja.exe" "-DBUILD_SHARED_LIBS=ON" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=C:/a/1/s/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x64-windows" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v143" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=C:/a/1/s/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=C:/a/1/s" "-D_VCPKG_INSTALLED_DIR=D:/installed" "-DVCPKG_MANIFEST_INSTALL=OFF" "-DCMAKE_DISABLE_SOURCE_CHANGES=ON" "-DINSTALL_DOCDIR:STRING=doc/Qt6/" "-DINSTALL_INCLUDEDIR:STRING=include/Qt6/" "-DINSTALL_DESCRIPTIONSDIR:STRING=share/Qt6/modules" "-DINSTALL_MKSPECSDIR:STRING=share/Qt6/mkspecs" "-DINSTALL_TRANSLATIONSDIR:STRING=translations/Qt6/" -G "Ninja" "-DCMAKE_BUILD_TYPE=Release" "-DCMAKE_INSTALL_PREFIX=D:/packages/qtinterfaceframework_x64-windows"
FAILED: ../CMakeCache.txt 
"D:/downloads/tools/cmake-3.22.2-windows/cmake-3.22.2-windows-i386/bin/cmake.exe" -E chdir "D:/buildtrees/qtinterfaceframework/x64-windows-rel" "D:/downloads/tools/cmake-3.22.2-windows/cmake-3.22.2-windows-i386/bin/cmake.exe" -S "D:/buildtrees/qtinterfaceframework/src/564c6bf227-ee28b9e83e.clean" "-DQT_USE_DEFAULT_CMAKE_OPTIMIZATION_FLAGS:BOOL=ON" "-DQT_BUILD_EXAMPLES:BOOL=OFF" "-DQT_BUILD_TESTS:BOOL=OFF" "-DQT_BUILD_BENCHMARKS:BOOL=OFF" "-DHOST_PERL:PATH=D:/downloads/tools/perl/5.32.1.1/perl/bin/perl.exe" "-DINSTALL_BINDIR:STRING=bin" "-DINSTALL_LIBEXECDIR:STRING=bin" "-DINSTALL_PLUGINSDIR:STRING=Qt6/plugins" "-DINSTALL_QMLDIR:STRING=Qt6/qml" "-DPython3_EXECUTABLE=D:/downloads/tools/python/python-3.10.2-x64/python.exe" "-DQT_SYNCQT:PATH=D:/installed/x64-windows/tools/Qt6/bin/syncqt.pl" "-DCMAKE_MAKE_PROGRAM=D:/downloads/tools/ninja/1.10.2-windows/ninja.exe" "-DBUILD_SHARED_LIBS=ON" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=C:/a/1/s/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x64-windows" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v143" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=C:/a/1/s/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=C:/a/1/s" "-D_VCPKG_INSTALLED_DIR=D:/installed" "-DVCPKG_MANIFEST_INSTALL=OFF" "-DCMAKE_DISABLE_SOURCE_CHANGES=ON" "-DINSTALL_DOCDIR:STRING=doc/Qt6/" "-DINSTALL_INCLUDEDIR:STRING=include/Qt6/" "-DINSTALL_DESCRIPTIONSDIR:STRING=share/Qt6/modules" "-DINSTALL_MKSPECSDIR:STRING=share/Qt6/mkspecs" "-DINSTALL_TRANSLATIONSDIR:STRING=translations/Qt6/" -G "Ninja" "-DCMAKE_BUILD_TYPE=Release" "-DCMAKE_INSTALL_PREFIX=D:/packages/qtinterfaceframework_x64-windows"
-- The CXX compiler identification is MSVC 19.31.31104.0
-- The C compiler identification is MSVC 19.31.31104.0
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: C:/Program Files/Microsoft Visual Studio/2022/Enterprise/VC/Tools/MSVC/14.31.31103/bin/Hostx64/x64/cl.exe - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: C:/Program Files/Microsoft Visual Studio/2022/Enterprise/VC/Tools/MSVC/14.31.31103/bin/Hostx64/x64/cl.exe - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Looking for pthread.h
-- Looking for pthread.h - not found
-- Found Threads: TRUE  
-- Performing Test HAVE_STDATOMIC
-- Performing Test HAVE_STDATOMIC - Success
-- Found WrapAtomic: TRUE  
-- CMAKE_BUILD_TYPE was set to: 'Release'
-- Check for feature set changes
-- Found Python3: D:/downloads/tools/python/python-3.10.2-x64/python.exe (found version "3.10.2") found components: Interpreter 
-- Performing Test Python3 virtualenv package
-- Performing Test Python3 virtualenv package - Success
-- Performing Test Python3 qface package
-- Checking for version:
-- Expected: 2.0.3
-- Condition: VERSION_GREATER_EQUAL
-- Got: 2.0.6

-- Performing Test Python3 qface package - Success
-- Found PkgConfig: D:/installed/x64-windows/tools/pkgconf/pkgconf.exe (found version "1.8.0") 
-- Checking for module 'taglib'
--   Found taglib, version 1.12
-- Running syncqt for module: 'QtInterfaceFramework' 
-- Running syncqt for module: 'QtIfRemoteObjectsHelper' 
-- Using source ifcodegen found at: D:/buildtrees/qtinterfaceframework/src/564c6bf227-ee28b9e83e.clean/src/tools/ifcodegen
-- Using source ifcodegen templates found at: D:/buildtrees/qtinterfaceframework/src/564c6bf227-ee28b9e83e.clean/src/tools/ifcodegen/templates
-- Running ifcodegen for D:/buildtrees/qtinterfaceframework/src/564c6bf227-ee28b9e83e.clean/src/ifvehiclefunctions/ifvehiclefunctions.qface with template frontend
CMake Error at src/interfaceframework/Qt6InterfaceFrameworkMacros.cmake:377 (message):
  Error while running the ifcodegen:

  D:/downloads/tools/python/python-3.10.2-x64/python.exe
  D:/buildtrees/qtinterfaceframework/src/564c6bf227-ee28b9e83e.clean/src/tools/ifcodegen/generate.py
  -T
  D:/buildtrees/qtinterfaceframework/src/564c6bf227-ee28b9e83e.clean/src/tools/ifcodegen/templates
  --template=frontend --force --module=QtIfVehicleFunctions
  D:/buildtrees/qtinterfaceframework/src/564c6bf227-ee28b9e83e.clean/src/ifvehiclefunctions/ifvehiclefunctions.qface
  D:/buildtrees/qtinterfaceframework/x64-windows-rel/src/ifvehiclefunctions

  Traceback (most recent call last):

    File "D:\buildtrees\qtinterfaceframework\src\564c6bf227-ee28b9e83e.clean\src\tools\ifcodegen\generate.py", line 40, in <module>
      from qface.generator import FileSystem, Generator
    File "D:\downloads\tools\python\python-3.10.2-x64\lib\site-packages\qface\generator.py", line 24, in <module>
      from .filters import get_filters
    File "D:\downloads\tools\python\python-3.10.2-x64\lib\site-packages\qface\filters.py", line 3, in <module>
      from .helper import qtcpp
    File "D:\downloads\tools\python\python-3.10.2-x64\lib\site-packages\qface\helper\qtcpp.py", line 5, in <module>
      from jinja2 import environmentfilter

  ImportError: cannot import name 'environmentfilter' from 'jinja2'
  (D:\downloads\tools\python\python-3.10.2-x64\lib\site-packages\jinja2\__init__.py)


Call Stack (most recent call first):
  src/interfaceframework/Qt6InterfaceFrameworkMacros.cmake:391 (qt6_ifcodegen_generate)
  src/ifvehiclefunctions/CMakeLists.txt:8 (qt_ifcodegen_generate)


-- Configuring incomplete, errors occurred!
See also "D:/buildtrees/qtinterfaceframework/x64-windows-rel/CMakeFiles/CMakeOutput.log".
See also "D:/buildtrees/qtinterfaceframework/x64-windows-rel/CMakeFiles/CMakeError.log".
[2/2] "D:/downloads/tools/cmake-3.22.2-windows/cmake-3.22.2-windows-i386/bin/cmake.exe" -E chdir "D:/buildtrees/qtinterfaceframework/x64-windows-dbg" "D:/downloads/tools/cmake-3.22.2-windows/cmake-3.22.2-windows-i386/bin/cmake.exe" -S "D:/buildtrees/qtinterfaceframework/src/564c6bf227-ee28b9e83e.clean" "-DQT_USE_DEFAULT_CMAKE_OPTIMIZATION_FLAGS:BOOL=ON" "-DQT_BUILD_EXAMPLES:BOOL=OFF" "-DQT_BUILD_TESTS:BOOL=OFF" "-DQT_BUILD_BENCHMARKS:BOOL=OFF" "-DHOST_PERL:PATH=D:/downloads/tools/perl/5.32.1.1/perl/bin/perl.exe" "-DINSTALL_BINDIR:STRING=bin" "-DINSTALL_LIBEXECDIR:STRING=bin" "-DINSTALL_PLUGINSDIR:STRING=Qt6/plugins" "-DINSTALL_QMLDIR:STRING=Qt6/qml" "-DPython3_EXECUTABLE=D:/downloads/tools/python/python-3.10.2-x64/python.exe" "-DQT_SYNCQT:PATH=D:/installed/x64-windows/tools/Qt6/bin/syncqt.pl" "-DCMAKE_MAKE_PROGRAM=D:/downloads/tools/ninja/1.10.2-windows/ninja.exe" "-DBUILD_SHARED_LIBS=ON" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=C:/a/1/s/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x64-windows" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v143" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=C:/a/1/s/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=C:/a/1/s" "-D_VCPKG_INSTALLED_DIR=D:/installed" "-DVCPKG_MANIFEST_INSTALL=OFF" "-DCMAKE_DISABLE_SOURCE_CHANGES=ON" "-DINSTALL_DOCDIR:STRING=../doc/Qt6/" "-DINSTALL_INCLUDEDIR:STRING=../include/Qt6/" "-DINSTALL_TRANSLATIONSDIR:STRING=../translations/Qt6/" "-DINSTALL_DESCRIPTIONSDIR:STRING=../share/Qt6/modules" "-DINSTALL_MKSPECSDIR:STRING=../share/Qt6/mkspecs" -G "Ninja" "-DCMAKE_BUILD_TYPE=Debug" "-DCMAKE_INSTALL_PREFIX=D:/packages/qtinterfaceframework_x64-windows/debug"
FAILED: ../../x64-windows-dbg/CMakeCache.txt 
"D:/downloads/tools/cmake-3.22.2-windows/cmake-3.22.2-windows-i386/bin/cmake.exe" -E chdir "D:/buildtrees/qtinterfaceframework/x64-windows-dbg" "D:/downloads/tools/cmake-3.22.2-windows/cmake-3.22.2-windows-i386/bin/cmake.exe" -S "D:/buildtrees/qtinterfaceframework/src/564c6bf227-ee28b9e83e.clean" "-DQT_USE_DEFAULT_CMAKE_OPTIMIZATION_FLAGS:BOOL=ON" "-DQT_BUILD_EXAMPLES:BOOL=OFF" "-DQT_BUILD_TESTS:BOOL=OFF" "-DQT_BUILD_BENCHMARKS:BOOL=OFF" "-DHOST_PERL:PATH=D:/downloads/tools/perl/5.32.1.1/perl/bin/perl.exe" "-DINSTALL_BINDIR:STRING=bin" "-DINSTALL_LIBEXECDIR:STRING=bin" "-DINSTALL_PLUGINSDIR:STRING=Qt6/plugins" "-DINSTALL_QMLDIR:STRING=Qt6/qml" "-DPython3_EXECUTABLE=D:/downloads/tools/python/python-3.10.2-x64/python.exe" "-DQT_SYNCQT:PATH=D:/installed/x64-windows/tools/Qt6/bin/syncqt.pl" "-DCMAKE_MAKE_PROGRAM=D:/downloads/tools/ninja/1.10.2-windows/ninja.exe" "-DBUILD_SHARED_LIBS=ON" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=C:/a/1/s/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x64-windows" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v143" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=C:/a/1/s/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=C:/a/1/s" "-D_VCPKG_INSTALLED_DIR=D:/installed" "-DVCPKG_MANIFEST_INSTALL=OFF" "-DCMAKE_DISABLE_SOURCE_CHANGES=ON" "-DINSTALL_DOCDIR:STRING=../doc/Qt6/" "-DINSTALL_INCLUDEDIR:STRING=../include/Qt6/" "-DINSTALL_TRANSLATIONSDIR:STRING=../translations/Qt6/" "-DINSTALL_DESCRIPTIONSDIR:STRING=../share/Qt6/modules" "-DINSTALL_MKSPECSDIR:STRING=../share/Qt6/mkspecs" -G "Ninja" "-DCMAKE_BUILD_TYPE=Debug" "-DCMAKE_INSTALL_PREFIX=D:/packages/qtinterfaceframework_x64-windows/debug"
-- The CXX compiler identification is MSVC 19.31.31104.0
-- The C compiler identification is MSVC 19.31.31104.0
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: C:/Program Files/Microsoft Visual Studio/2022/Enterprise/VC/Tools/MSVC/14.31.31103/bin/Hostx64/x64/cl.exe - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: C:/Program Files/Microsoft Visual Studio/2022/Enterprise/VC/Tools/MSVC/14.31.31103/bin/Hostx64/x64/cl.exe - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Looking for pthread.h
-- Looking for pthread.h - not found
-- Found Threads: TRUE  
-- Performing Test HAVE_STDATOMIC
-- Performing Test HAVE_STDATOMIC - Success
-- Found WrapAtomic: TRUE  
-- CMAKE_BUILD_TYPE was set to: 'Debug'
-- Check for feature set changes
-- Found Python3: D:/downloads/tools/python/python-3.10.2-x64/python.exe (found version "3.10.2") found components: Interpreter 
-- Performing Test Python3 virtualenv package
-- Performing Test Python3 virtualenv package - Success
-- Performing Test Python3 qface package
-- Checking for version:
-- Expected: 2.0.3
-- Condition: VERSION_GREATER_EQUAL
-- Got: 2.0.6

-- Performing Test Python3 qface package - Success
-- Found PkgConfig: D:/installed/x64-windows/tools/pkgconf/pkgconf.exe (found version "1.8.0") 
-- Checking for module 'taglib'
--   Found taglib, version 1.12
-- Running syncqt for module: 'QtInterfaceFramework' 
-- Running syncqt for module: 'QtIfRemoteObjectsHelper' 
-- Using source ifcodegen found at: D:/buildtrees/qtinterfaceframework/src/564c6bf227-ee28b9e83e.clean/src/tools/ifcodegen
-- Using source ifcodegen templates found at: D:/buildtrees/qtinterfaceframework/src/564c6bf227-ee28b9e83e.clean/src/tools/ifcodegen/templates
-- Running ifcodegen for D:/buildtrees/qtinterfaceframework/src/564c6bf227-ee28b9e83e.clean/src/ifvehiclefunctions/ifvehiclefunctions.qface with template frontend
CMake Error at src/interfaceframework/Qt6InterfaceFrameworkMacros.cmake:377 (message):
  Error while running the ifcodegen:

  D:/downloads/tools/python/python-3.10.2-x64/python.exe
  D:/buildtrees/qtinterfaceframework/src/564c6bf227-ee28b9e83e.clean/src/tools/ifcodegen/generate.py
  -T
  D:/buildtrees/qtinterfaceframework/src/564c6bf227-ee28b9e83e.clean/src/tools/ifcodegen/templates
  --template=frontend --force --module=QtIfVehicleFunctions
  D:/buildtrees/qtinterfaceframework/src/564c6bf227-ee28b9e83e.clean/src/ifvehiclefunctions/ifvehiclefunctions.qface
  D:/buildtrees/qtinterfaceframework/x64-windows-dbg/src/ifvehiclefunctions

  Traceback (most recent call last):

    File "D:\buildtrees\qtinterfaceframework\src\564c6bf227-ee28b9e83e.clean\src\tools\ifcodegen\generate.py", line 40, in <module>
      from qface.generator import FileSystem, Generator
    File "D:\downloads\tools\python\python-3.10.2-x64\lib\site-packages\qface\generator.py", line 24, in <module>
      from .filters import get_filters
    File "D:\downloads\tools\python\python-3.10.2-x64\lib\site-packages\qface\filters.py", line 3, in <module>
      from .helper import qtcpp
    File "D:\downloads\tools\python\python-3.10.2-x64\lib\site-packages\qface\helper\qtcpp.py", line 5, in <module>
      from jinja2 import environmentfilter

  ImportError: cannot import name 'environmentfilter' from 'jinja2'
  (D:\downloads\tools\python\python-3.10.2-x64\lib\site-packages\jinja2\__init__.py)


Call Stack (most recent call first):
  src/interfaceframework/Qt6InterfaceFrameworkMacros.cmake:391 (qt6_ifcodegen_generate)
  src/ifvehiclefunctions/CMakeLists.txt:8 (qt_ifcodegen_generate)


-- Configuring incomplete, errors occurred!
See also "D:/buildtrees/qtinterfaceframework/x64-windows-dbg/CMakeFiles/CMakeOutput.log".
See also "D:/buildtrees/qtinterfaceframework/x64-windows-dbg/CMakeFiles/CMakeError.log".
ninja: build stopped: subcommand failed.
```

**Additional context**
Add any other context about the problem here, such as what you have already tried to resolve the issue.
CC: @Neumann-A

--- a/scripts/ci.baseline.txt
+++ b/scripts/ci.baseline.txt
@@ -1023,6 +1023,9 @@ qt5-base:arm64-windows=fail
 # qtwebengine:x64-windows has an ICE in VS2022
 qtwebengine:x64-windows=fail
 
+# upstream bug, see https://github.com/microsoft/vcpkg/issues/23766
+qtinterfaceframework:x64-windows=fail
+
 # Skip deprecated Qt module
 # (remove after 1 year or longer due to vcpkg upgrade not handling removed ports correctly)
 qt5-canvas3d:x64-linux=skip

